src/model.js

```diff
@@ -0,0 +1,58 @@
+export default class Model {
+  constructor() {
+    this.tasks = JSON.parse(localStorage.getItem('todos')) || [];
+  }
+
+  setStorage = (todos) => {
+    // this.tasks.forEach(element => {
+    localStorage.setItem('todos', JSON.stringify(todos));
+    // });
+  }
+
+  bindChange = (callback) => {
+    this.change = callback;
+  }
+
+  update(tasks) {
+    this.change(tasks);
+    this.setStorage(tasks);
+  }
```

Comment:
- [ ] For this milestone, you are required to add some functionality to the application to make it interactive. The user should be able to mark task completion by selecting the corresponding checkbox. The updated tasks list will be stored in local storage.
- For your project, your updated task list is not stored in the localStorage. Give this a fix.

Reply:
Thank you. My code broke after I corrected some linter errors. I will fix it now.